docs/unified-test-documentation/dasharo-security/20F-me-neuter.md

```diff
@@ -13,7 +13,41 @@
 1. Proceed with the
     [Generic test setup: OS boot from disk](../../generic-test-setup#os-boot-from-disk).
 
-## MNE001.001 Intel ME mode option is available and has the correct default state
+## MNE001.001 Check Intel ME version (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to check Intel ME version, should be the same as in
+the release notes.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Open a terminal window and run the following command:
+
+    ```bash
+    cat /sys/class/mei/mei0/fw_ver
+    ```
+
+1. Note the results.
+
+**Expected result**
+
+The Intel ME version should be the same as in the release notes.
```

Comment:
It would be good to add some examples of what the output from the command looks like and which of the resulting values is correct.

Reply:
It is good form for example?

Reply:
I change test description to:
`This test aims to verify the Intel ME version, should be the same as in
the release notes.`